gnomad/utils/filtering.py

```diff
@@ -533,40 +533,46 @@ def split_vds_by_strata(
     }
 
 
-def filter_freq_by_meta(
-    freq_expr: hl.expr.ArrayExpression,
-    freq_meta_expr: hl.expr.ArrayExpression,
+def filter_arrays_by_meta(
+    meta_expr: hl.expr.ArrayExpression,
+    meta_indexed_exprs: Dict[str, hl.expr.ArrayExpression],
```

Comment:
I have one small suggestion. Just to make other code that uses this a little cleaner in the case that only one array needs to be changed, we could have this be:

```suggestion
    meta_indexed_exprs: Union[Dict[str, hl.expr.ArrayExpression], hl.expr.ArrayExpression],
```

Then also update the output type, and add in 

```
if isinstance(meta_indexed_exprs, hl.expr.ArrayExpression):
    meta_indexed_exprs = {"_tmp": meta_indexed_exprs}
```

And then right before the return add:
```
if "_tmp" in meta_indexed_exprs:
    meta_indexed_exprs = meta_indexed_exprs["_tmp"]
```

Reply:
Good call, added it!